dev-ftp/README.md

```diff
@@ -0,0 +1,41 @@
+# Development FTP server
+
+Simple local FTP server to help test the `ftp-watcher` service in
+development.
+
+To use this development FTP server, add the following lines to your
+`/etc/gu/ftp-watcher.properties` file:
+
+```
+ftp.active=true
+ftp.host=localhost
+ftp.port=41756
+ftp.user=any
+ftp.password=any
+```
+
+Install the necessary dependencies:
+
+```
+$ gem install ftpd
```

Comment:
Yeah I thought it's a bit overkill, can add if people disagree

/cc @kenoir who knows about Ruby things